packages/next/build/index.ts

```diff
@@ -271,7 +271,7 @@ export default async function build(dir: string, conf = null): Promise<void> {
       actualPage + '.js'
     )
 
-    if (!page.match(/(_app|_error|_document)/)) {
+    if (!page.match(/(_app|_error|_document|^\/api)/)) {
```

Comment:
This is missing `/^` so it'll match `_app` anywhere in the string.

Reply:
Updated